commcare_connect/opportunity/utils/completed_work.py

```diff
@@ -1,84 +1,182 @@
+from collections import defaultdict
+
+from django.db.models import Prefetch, Sum
+
 from commcare_connect.opportunity.models import (
     CompletedWork,
     CompletedWorkStatus,
+    DeliverUnit,
     OpportunityAccess,
     Payment,
+    PaymentUnit,
+    UserVisit,
     VisitReviewStatus,
     VisitValidationStatus,
 )
 
 
+def get_completed_work_completed_approved_count(completed_works):
+    counts = defaultdict(lambda: {"completed": 0, "approved": 0})
+
+    completed_works = completed_works.prefetch_related(
+        Prefetch("uservisit_set", queryset=UserVisit.objects.all()), "payment_unit"
```

Comment:
why is the queryset necessary?

Reply:
This is redundant. I have removed the queryset since doing a preftech for uservisit set already does the same. Also removed the payment_unit prefetch here as that is also not needed.
[f950ece](https://github.com/dimagi/commcare-connect/pull/529/commits/f950ece33185ea52aab20428934068aeeee3bdfa)